### DISCUSSION TITLE
Is this project open source?

### DISCUSSION THREAD

**Discussion by opratrx** [qu8n/pennplanner#5] [Q&A] [answered]
Hey qu8n,

Saw this on reddit, this is truly amazing. Just wondering if this is open source? If so, can you provide the license? 

Thanks!

**Comment by qu8n** [comment]
@opratrx Thank you, I really appreciate it. 

I'm still in the process of evaluating which open source license to use. Would you be open to clarify what your intended use is for this project?

**Reply by opratrx** [reply]
@qu8n Thanks for the speedy response! I just want to be able to host a variation for the CS program at WGU. We have a student ran discord full of colleagues who could find this very helpful with planning.

**Reply by qu8n** [reply, marked as answer]
@opratrx Just added the MIT license to the project, which should make it easy for you to fork and work on top of it. I just ask that you credit/link back to this repo somewhere in your app if you have room for it.

Btw, I heard great things about WGU's CS program. Best of luck with your app!

**Comment by opratrx** [comment]
@qu8n Thanks you so much!! I'll definitely add credit and make sure it adheres to the license as well. I appreciate your help with my query. 😊
